medical_pathology/demo/medical_pathology_category_demo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+
+<odoo>
+
+    <record id="medical_pathology_category_1" model="medical.pathology.category">
```

Comment:
Keep in mind that major restructuring is planned for v11 too. We're going to invalidate a lot of stuff